cdbot/cogs/cyber.py

```diff
@@ -360,6 +360,73 @@ def __init__(self, bot: Bot):
         await ctx.send(f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, "
                        f"{len(true_r.members)} have also completed Essentials and Assess.")
 
+    @command()
+    async def elitecountdown(self, ctx: Context, event_name: str = None, event_level: str = None):
+        """
+        Gets the time until elite, and shows in an embed
+        """
+
+        # Validate and normalise inputs
+        if event_name:
+            event_name = event_name.capitalize()
+            if not (event_name in ELITE_LOCATIONS):
```

Comment:
I really believe this line should be removed in favour of a bigger `ELITE_ALISES` dictionnary.
Look at this code from another bot:
https://github.com/python-discord/seasonalbot/blob/9daf007c4094b8ee09b4779f78ac9122d0ca4e7d/bot/seasons/pride/pride_avatar.py#L13